### PR TITLE
Add star-rating.js

### PR DESCRIPTION
**Pull request by pryley**

### CODE REVIEW

**Comment by pineapplemachine** [comment]
The `source` attribute for star-rating should be an array containing paths to both the JavaScript and the needed css. Can you make this change? If you do that, it looks like this should be good to go.

File: data.js [hunk 1]
```diff
@@ -38,6 +38,14 @@ module.exports = [{
     url: "https://github.com/eorroe/NodeList.js",
     source: "https://raw.githubusercontent.com/eorroe/NodeList.js/master/NodeList.js"
   },
+  {
+    name: "Star Rating",
+    github: "geminilabs/star-rating.js",
+    tags: ["rating", "stars", "star-rating"],
+    description: "Transforms a SELECT element into a dynamic star rating control",
+    url: "https://geminilabs.github.io/star-rating.js/",
+    source: "https://raw.githubusercontent.com/geminilabs/star-rating.js/master/src/star-rating.js"
```
Comment (pineapplemachine) [inline data.js]: Should be an array of paths for both the needed JS and CSS.

Comment (pryley) [reply data.js]: Done

**Comment by pineapplemachine** [comment]
The data.json entry looks good now. Thank you for contributing!

**Pull request merged**